src/modules/dashboard/components/Task/Task.tsx

```diff
@@ -124,36 +122,38 @@ const Task = ({
 
   const { walletAddress } = useLoggedInUser();
 
-  const { data: colonyAddress } = useDataFetcher(
-    colonyAddressFetcher,
-    [colonyName],
-    [colonyName],
-  );
+  const { data } = useTaskQuery({
```

Comment:
It's good to know that this exists as some sort of stop gap solution. 5 seconds is probably enough